news-bundle/src/Controller/NewsFeedController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\Controller;
```

Comment:
I think we moved page controllers to a subfolder

```suggestion
namespace Contao\NewsBundle\Controller\Page;
```